data/model/test/test_org_mirror.py

```
@@ -674,3 +681,745 @@ def test_update_org_mirror_config_preserves_unchanged_fields(self, initialized_d
         assert updated.visibility.name == "public"
         assert updated.repository_filters == filters
         assert updated.skopeo_timeout == 300
+
+
+class TestGetEligibleOrgMirrorRepos:
+    """Tests for get_eligible_org_mirror_repos function."""
+
+    def test_no_repos_returns_empty(self, initialized_db):
```

Ruff (0.14.13)

Unused method argument: `initialized_db`

(ARG002)